pilot/cmd/pilot-discovery/main.go

```diff
@@ -138,6 +138,9 @@ func init() {
 	discoveryCmd.PersistentFlags().BoolVar(&serverArgs.DiscoveryOptions.EnableCaching, "discoveryCache", true,
 		"Enable caching discovery service responses")
 
+	discoveryCmd.PersistentFlags().StringVar(&serverArgs.MixerIdentity, "MixerIdentity", "",
+		"value to override Mixer SAN delivered to pilots proxy")
```

Comment:
Why can't the Mixer's identity come from the annotation of Mixer's pods? Is there's a racing issue?

Reply:
The MixerSAN from the env model value is written during the server start.  It looks like it's only writen for tests as an empty value and never read.  I think, this can be delete along with the MixerSAN from the env model.  The value disseminated from pilot to the proxies uses the values from the pod.